src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -47,74 +61,57 @@ def create(dataset, target, feature = None, model = 'resnet-50',
         values is mapped to 0 and 1 respectively with 0 being the base class
         and 1 being the reference class. Use `model.classes` to retrieve
         the order in which the classes are mapped.
-
     feature : string, optional
         indicates that the SFrame has only column of Image type and that will
         Name of the column containing the input images. 'None' (the default)
         indicates the only image column in `dataset` should be used as the
         feature.
-
     model : string optional
         Uses a pretrained model to bootstrap an image classifier:
-
            - "resnet-50" : Uses a pretrained resnet model.
                            Exported Core ML model will be ~90M.
-
            - "squeezenet_v1.1" : Uses a pretrained squeezenet model.
                                  Exported Core ML model will be ~4.7M.
-
            - "VisionFeaturePrint_Screen": Uses an OS internal feature extractor.
                                           Only on available on iOS 12.0+,
                                           macOS 10.14+ and tvOS 12.0+.
                                           Exported Core ML model will be ~41K.
-
         Models are downloaded from the internet if not available locally. Once
         downloaded, the models are cached for future use.
-
     validation_set : SFrame, optional
```

Comment:
You should add documentation for each new parameter. This text can probably largely be copied from [the docstring for logistic regression's create function](https://github.com/apple/turicreate/blob/86c65447504be4e4bdd3378d88957c6f1b0d4d63/src/unity/python/turicreate/toolkits/classifier/logistic_classifier.py#L89).